src/main/java/com/snowflake/kafka/connector/internal/ingestsdk/StreamingClientManager.java

```diff
@@ -72,25 +87,42 @@ public void createAllStreamingClients(
     this.maxTasks = maxTasks;
 
     int clientCount = (int) Math.ceil((double) maxTasks / (double) numTasksPerClient);
-    LOGGER.info(
-        "Creating {} clients for {} tasks with max {} tasks per client",
-        clientCount,
-        maxTasks,
-        numTasksPerClient);
 
     Properties clientProperties = new Properties();
     clientProperties.putAll(
         StreamingUtils.convertConfigForStreamingClient(new HashMap<>(connectorConfig)));
 
+    Map<String, Object> parameterOverrides = new HashMap<>();
+    // default file type is PARQUET defined in Ingest SDK dependency
+    String snowpipeStreamingBdecFileType =
+        connectorConfig.getOrDefault(
+            SnowflakeSinkConnectorConfig.SNOWPIPE_STREAMING_FILE_TYPE,
+            SnowpipeStreamingFileType.PARQUET.toString());
+    if (snowpipeStreamingBdecFileType.equalsIgnoreCase(
```

Comment:
i think this doesn't have code coverage? didn't run it locally, resolve if it is covered

Reply:
Have a test case, resolving it.